detox/e2e/test/smoke_test/server_login.e2e.ts

```diff
@@ -79,11 +79,11 @@ describe('Smoke Test - Server Login', () => {
 
         // # Go back to first server, open server list screen, swipe left on second server and tap on logout option
         await ServerListScreen.open();
-        await ServerListScreen.serverListScreen.swipe('up');
+        await ServerListScreen.serverListTitle.swipe('up', 'fast', 0.1, 0.5, 0.3);
```

Comment:
The change here and the other one failed on me in iOS due to bottom sheet not being pulled up fully causing the "Log Out" button to be partially covered (per Detox visibility check).
<img width="975" alt="Screenshot 2023-02-27 at 6 11 45 PM" src="https://user-images.githubusercontent.com/5334504/221539222-9d308736-c820-47cb-beaa-27238ec5a1ef.png">
<img width="416" alt="Screenshot 2023-02-27 at 6 11 29 PM" src="https://user-images.githubusercontent.com/5334504/221540197-18fe576e-c5c2-4c0c-bdfb-5457c93be79d.png">

However, the original code passed in iOS.
<img width="925" alt="Screenshot 2023-02-27 at 6 25 31 PM" src="https://user-images.githubusercontent.com/5334504/221539982-95d6c581-a28e-4105-92f4-ec61f7a4eca3.png">

How about change the swipe up action to Android only?